FrejaEidClient/src/main/java/com/verisec/frejaeid/client/client/impl/OrganisationIdClient.java

```diff
@@ -100,6 +103,15 @@ public void delete(DeleteOrganisationIdRequest deleteOrganisationIdRequest) thro
         LOG.debug("Successfully deleted organisation ID identifier {}.", deleteOrganisationIdRequest.getIdentifier());
     }
 
+    @Override
+    public List<OrganisationIdUserInfo> getAllUsers(GetAllOrganisationIdUsersRequest getAllOrganisationIdUsersRequest) throws FrejaEidClientInternalException, FrejaEidException {
+        requestValidationService.validateGetAllOrganisationIdUsersRequest(getAllOrganisationIdUsersRequest);
+        LOG.debug("Getting information about users with organisation ID.");  
```

Comment:
Would it be good to log relyingPartyId for which this request is made?

Reply:
Relying party id can be null. Do we want to log it? If yes, should we log relying party id on other places (authentication, sign...)?

Reply:
As we agreed, will leave as it is.